### PR TITLE
[fixed] Respect onClick on MenuItem

### PR DESCRIPTION
**Pull request by taion**
Fixes https://github.com/react-bootstrap/react-router-bootstrap/issues/112.

Same caveat as on #1413.

### CODE REVIEW

File: src/MenuItem.js [hunk 1]
```diff
@@ -35,6 +37,8 @@ export default class MenuItem extends React.Component {
       );
     }
 
+    const {className, style, href, onClick, ...props} = this.props;
```
Comment (mathieumg) [inline src/MenuItem.js]: I imagine `href = '',` could have worked here?

**Comment by mathieumg** [comment]
LGTM

File: src/MenuItem.js (hunk 1 above)
Comment (taion) [inline src/MenuItem.js]: Not quite - the destructuring default only works on `undefined`, where as the `href || ''` replaces any falsy values. I don't know if that part matters, but it'd be an additional behavior change I didn't want to introduce.

Comment (mathieumg) [inline src/MenuItem.js]: Alright, that's a subtle difference but it makes sense, thanks for clarifying!

**Comment by jchapple** [comment]
Cheers for getting this fixed, @taion. :+1:

**Pull request merged**